apps/redi-talent-pool/src/components/organisms/company-profile-editables/EditableJobPostings.tsx

```diff
@@ -185,6 +157,100 @@ interface ModalFormProps {
   setIsEditing: (boolean) => void
 }
 
+function JobListingCards({ jobListings, startEditing }) {
+  const queryClient = useQueryClient()
+
+  const renderTimestamp = (expiresAt) =>
+    `Expires ${formatDistance(new Date(expiresAt), new Date(), {
+      addSuffix: true,
+    })}`
+
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false)
+  const handleDeleteModalClose = useCallback(
+    () => setIsDeleteModalOpen(false),
+    []
+  )
+  const deleteMutation = useTpJobListingDeleteMutation()
+  const handleDelete = useCallback(
+    (id: string) => {
+      handleDeleteModalClose()
+      deleteMutation.mutate(
+        { input: { id } },
+        {
+          onSuccess: () => {
+            queryClient.invalidateQueries()
+          },
+        }
+      )
+    },
+    [deleteMutation, handleDeleteModalClose, queryClient]
+  )
+
+  const copyUrl = useCallback((id: string) => {
+    // Keep in sync with routes__logged-in.tsx:
+    // TODO: use a shared function/constant
+    window.navigator.clipboard.writeText(
+      TALENT_POOL_URL + '/app/job-listing/' + id
+    )
+    showNotification('Link copied to clipboard')
+  }, [])
+
+  return (
+    <Columns>
+      {jobListings?.map((jobListing) => (
+        <Columns.Column size={12} key={jobListing.id}>
+          <JobListingCard
+            key={jobListing.id}
```

Comment:
Do we need this key, having the key in the Column already?

Reply:
Nope. Removed.